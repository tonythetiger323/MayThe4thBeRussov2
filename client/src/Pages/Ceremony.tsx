import * as React from 'react'
import CeremonyCard from '../Components/CeremonyCard'
import Grid from '@material-ui/core/Grid'

class Ceremony extends React.Component {
  render() {
    return (
      <div>
        <Grid container direction='column' alignContent='center'>
          <CeremonyCard />
        </Grid>
      </div>
    )
  }
}

export default Ceremony
