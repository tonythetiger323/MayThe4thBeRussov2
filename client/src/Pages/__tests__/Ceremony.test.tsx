import * as React from 'react'
import { shallow } from 'enzyme'
import Ceremony from '../Ceremony'

describe('Ceremony Page', () => {
  let wrapper: any
  beforeEach(() => (wrapper = shallow(<Ceremony />)))

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
