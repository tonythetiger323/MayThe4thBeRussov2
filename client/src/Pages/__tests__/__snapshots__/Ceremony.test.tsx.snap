// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ceremony Page should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Ceremony />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Countdown
          date="May 4, 2019 09:00:00"
        />,
        <WithStyles(CeremonyCard) />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "date": "May 4, 2019 09:00:00",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Countdown
            date="May 4, 2019 09:00:00"
          />,
          <WithStyles(CeremonyCard) />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "date": "May 4, 2019 09:00:00",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
